### PR TITLE
upgrades symfony and doctrine

### PR DESCRIPTION
**Pull request by Elorfin**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes

- Upgrades symfony from 4 to 5
- Upgrades dbal from 2 to 3

### CODE REVIEW

**Comment by chalasr** [comment]
Hey 👋 So we definitely drop the SAML integration? (I started working on this and took the path of forking to keep the feature available, but if we don't need it that's a big win!)

**Comment by Elorfin** [comment]
Hi !
I dropped it for 2 reasons : 
- I want to be able to "stabilize" a version without having to fork and upgrade the underlying libraries. I already need to fix doctrine migrations and an edge case in voters.
- We will move the SAML feature in a private package.

File: src/main/migration/Command/AbstractMigrateCommand.php [hunk 1]
```diff
@@ -40,10 +40,10 @@ protected function configure()
 but you can specify a target using the <info>--target</info> option:
 
     <info>%command.name% AcmeFooBundle --target=YYYYMMDDHHMMSS</info>
-    <info>%command.name% AcmeFooBundle --target=farthest</info>
+    <info>%command.name% AcmeFooBundle --target=lastest</info>
```
Comment (LaurentGruber) [inline src/main/migration/Command/AbstractMigrateCommand.php:43]: il n'y aurait pas une typo isi lastest au lieu de latest (idem ligne 46) ?

**Comment by chalasr** [comment]
For the doctrine/dbal CI error:
```diff
- Doctrine\DBAL\Driver\PDOMySql\Driver
+ Doctrine\DBAL\Driver\PDO\MySQL\Driver
```

**Pull request merged**